buildchain/buildchain/salt_tree.py

```diff
@@ -81,158 +81,147 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
-    Path('salt/metalk8s/kubeadm/init/certs/apiserver-kubelet-client.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/ca.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/etcd-ca.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/etcd-deploy-ca-cert.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/etcd-healthcheck-client.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/etcd-peer.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/etcd-server.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/front-proxy-ca.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/front-proxy-deploy-ca-cert.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/front-proxy-client.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/init.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/installed.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/sa.sls'),
-    Path('salt/metalk8s/kubeadm/init/certs/sa-deploy-pub-key.sls'),
-
-    Path('salt/metalk8s/kubeadm/init/control-plane/apiserver.sls'),
-    Path('salt/metalk8s/kubeadm/init/control-plane/controller-manager.sls'),
-    Path('salt/metalk8s/kubeadm/init/control-plane/files/htpasswd'),
-    Path('salt/metalk8s/kubeadm/init/control-plane/files/manifest.yaml'),
-    Path('salt/metalk8s/kubeadm/init/control-plane/init.sls'),
-    Path('salt/metalk8s/kubeadm/init/control-plane/scheduler.sls'),
-
-    Path('salt/metalk8s/kubeadm/init/etcd/init.sls'),
-    Path('salt/metalk8s/kubeadm/init/etcd/files/manifest.yaml'),
-    Path('salt/metalk8s/kubeadm/init/etcd/local.sls'),
-
-    Path('salt/metalk8s/kubeadm/init/init.sls'),
-
-    Path('salt/metalk8s/kubeadm/init/kubelet-start/configured.sls'),
-    Path('salt/metalk8s/kubeadm/init/kubelet-start/files/kubeadm.env'),
+    Path('salt/metalk8s/_internal/certs/init.sls'),
+    Path('salt/metalk8s/_internal/certs/installed.sls'),
+    Path('salt/metalk8s/_internal/certs/removed.sls'),
+
+    Path('salt/metalk8s/_internal/files/control-plane-manifest.yaml'),
+
+    Path('salt/metalk8s/_internal/init.sls'),
+
+    Path('salt/metalk8s/_internal/preflight/init.sls'),
+    Path('salt/metalk8s/_internal/preflight/mandatory.sls'),
+    Path('salt/metalk8s/_internal/preflight/recommended.sls'),
+
+    Path('salt/metalk8s/_internal/python-kubernetes/init.sls'),
+    Path('salt/metalk8s/_internal/python-kubernetes/installed.sls'),
+    Path('salt/metalk8s/_internal/python-kubernetes/removed.sls'),
+
+    Path('salt/metalk8s/kubernetes/apiserver/certs/etcd-client.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/certs/front-proxy-client.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/certs/init.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/certs/kubelet-client.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/certs/server.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/files/htpasswd'),
+    Path('salt/metalk8s/kubernetes/apiserver/init.sls'),
+    Path('salt/metalk8s/kubernetes/apiserver/installed.sls'),
```

Comment:
I guess this will need an `absent` as well (etc. etc. etc.)

Reply:
Yes but in another PR I think it's part of #986